src/linear_algebra/exterior_algebra.lean

```diff
@@ -156,4 +166,59 @@ begin
   simp only [h],
 end
 
+@[simp]
+lemma ι_add_mul_swap (x y : M) : ι R x * ι R y + ι R y * ι R x = 0 :=
+calc _ = ι R (x + y) * ι R (x + y) : by simp [mul_add, add_mul]
+   ... = _ : ι_square_zero _
+
+lemma ι_mul_prod_list {n : ℕ} (f : fin n → M) (i : fin n) :
+  ι R (f i) * (list.of_fn $ λ i, ι R $ f i).prod = 0 :=
+begin
+  induction n with n hn,
+  { exact i.elim0, },
+  { rw [list.of_fn_succ, list.prod_cons, ←mul_assoc],
+    by_cases h : i = 0,
+    { rw [h, ι_square_zero, zero_mul], },
+    { replace hn := congr_arg ((*) (ι R (f 0))) (hn (λ i, f $ fin.succ i) (i.pred h)),
+      simp only at hn,
+      rw [fin.succ_pred, ←mul_assoc, mul_zero] at hn,
+      rw ← zero_add (_ * _),
+      conv_lhs {rw ← hn},
+      rw [← add_mul, ι_add_mul_swap, zero_mul], } }
+end
+
+variables (R)
+/-- The product of `n` terms of the form `ι R m` is an alternating map.
+
+This is a special case of `multilinear_map.mk_pi_algebra_fin` -/
+def ι_multi (n : ℕ) :
```

Comment:
I'm not really sure what a good name for this is. `wedge` is another option, but it's weird that we use `*` as the symbol for a product that we call a wedge product.

Reply:
The name looks fine to me right now, since it's the (alternating) multilinear version of `ι`.